src/burn/engine/variable.cpp

```diff
@@ -889,6 +896,17 @@ extern "C" HRESULT VariableDeserialize(
     __inout SIZE_T* piBuffer
     )
 {
+    return VariableDeserializeEx(pVariables, TRUE, pbBuffer, cbBuffer, piBuffer);
+}
+
+extern "C" HRESULT VariableDeserializeEx(
```

Comment:
It's OK to change function signatures in the engine. There's only one instance of VariableDeserialize left.